webapp/src/lib/server/databaseBackend.ts

```diff
@@ -0,0 +1,26 @@
+import { supabase } from "$lib/server/supabaseBackendClient";
+
+export async function createCard(card: Card) {
```

Comment:
Do we want to define the Promise everywhere? I did not do it in my functions, so we should add this to them?